recipes/oyaml/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "oyaml" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5064e85aa38ec481b1c49e4941957c6c5e1961291ce276b064c5d6a85d833ec5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - pyyaml
```

Comment:
I do not think you need `pyyaml` as a `host` requirement since it isn't necessary to build the package.